packages/amplify-category-api/provider-utils/awscloudformation/sync-conflict-handler-assets/sync-conflict-handler-package.json.ejs

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "<%= props.functionName %>",
+  "version": "1.0.0",
+  "description": "Lambda function generated by Amplify for conflict handling",
+  "main": "index.js",
+  "license": "Apache-2.0"
```

Comment:
Do we want to enforce a license on generated code on the customer's behalf? Can it be empty instead?